tests/unit_tests/test_wave_06.py

```diff
@@ -143,16 +135,17 @@ def test_swap_best_by_category_reordered():
         their_priority="Decor"
     )
 
-    raise Exception("Complete this test according to comments below.")
-    # *********************************************************************
-    # ****** Complete Assert Portion of this test **********
-    # *********************************************************************
-    # Assertions should check:
-    # - That result is truthy
-    # - That tai and jesse's inventories are the correct length
-    # - That all the correct items are in tai and jesse's inventories, and that the items that were swapped are not there
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
+    assert item_f in tai.inventory
+    assert item_b in tai.inventory
+    assert item_a in tai.inventory
+    assert item_c in jesse.inventory
+    assert item_d in jesse.inventory
+    assert item_e in jesse.inventory
+    assert item_f not in jesse.inventory
+    assert item_c not in tai.inventory
```

Comment:
We could remove these last two asserts. Since we check that the length of the inventories is 3, and we check for the presence of 3 items in each of the inventories, we have already confirmed that these items are not in the lists mentioned.